ngraph/core/src/op/split.cpp

```diff
@@ -142,6 +138,9 @@ namespace split
 bool op::v1::Split::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
 {
     NGRAPH_OP_SCOPE(v1_Split_evaluate);
+    NGRAPH_CHECK(this,
```

Comment:
It is not correct. In this case `NGRAPH_CHECK` doesn't check your condition, it checks: `!this`
Please use:
```
NODE_VALIDATION_CHECK(this, validate_host_tensor_vector(outputs, m_num_splits) &&
                     validate_host_tensor_vector(inputs, 2));
```
or
```
NGRAPH_CHECK(validate_host_tensor_vector(outputs, m_num_splits) &&
                     validate_host_tensor_vector(inputs, 2));
```

Reply:
Good catch. Done